HLTrigger/HLTfilters/src/HLTL1TSeed.cc

```diff
@@ -577,14 +577,15 @@ bool HLTL1TSeed::seedsL1TriggerObjectMaps(edm::Event& iEvent,
 
         for (std::vector<SingleCombInCond>::const_iterator itComb = (*condComb).begin(); itComb != (*condComb).end(); itComb++) {
 
+            LogTrace("HLTL1TSeed")
+            << setw(15) << "\tnew combination" << endl;
+
+            size_t iType = 0; 
```

Comment:
hi @rekovic , could you change this into 

```c++
// the index of the object type is the same as the index of the object
size_t iType = std::distance((*itComb).begin(), itObject)
```

Just to avoid that the next time we look at the code, we forget about this and de-sync the two again :-(